ci/concourse/build-rust.sh

```diff
@@ -1,29 +1,25 @@
 #!/bin/bash
-set -euxo pipefail
 
-. "$(dirname $0)"/env.sh
+set -euo pipefail
```

Comment:
I tend to like -x in CI for showing exactly where errors occurred, fwiw

Reply:
I think we're ok with this for now, mainly because the CI can be debugged directly with `fly hijack` if we run into issues. Just helps keep the logs clean, it can be tough to read otherwise